examples/text-generation/README.md

```diff
@@ -198,6 +198,24 @@ python ../gaudi_spawn.py --use_deepspeed --world_size 8 run_generation.py \
 > --bf16
 > ```
 
+To run Mamba-130m inference on 1 Gaudi2 card, use the following command:
+```bash
+python run_generation.py \
+--model_name_or_path state-spaces/mamba-130m-hf \
+--max_input_tokens 128 \
+--max_new_tokens 128 \
+--bf16 \
+--use_hpu_graphs \
+--use_kv_cache \
+--batch_size 1024 \
+```
+We need to download two shared libraries and set two env variables for mamba, for 1.19 version use *_119 so files. For different release, use different set of these so files.
+```bash
+wget https://huggingface.co/Habana/mamba/resolve/main/hpu_custom_pscan_all.cpython-310-x86_64-linux-gnu_119.so
+wget https://huggingface.co/Habana/mamba/resolve/main/libcustom_tpc_perf_lib_119.so
+export HABANA_CUSTOM_OP_DIR=/path/of/hpu_custom_pscan_all.cpython-310-x86_64-linux-gnu_119.so located
+export GC_KERNEL_PATH=/path/to/libcustom_tpc_perf_lib_119.so:$GC_KERNEL_PATH
```

Comment:
This should be done automatically in the script if the model type is Mamba

Reply:
The test did the automatically download, but for users want to run the mamba, they can just download the so files form the Habana/mamba card.

Reply:
I also remove the wget and use hf_hub_download in the test script.

Reply:
I think this will discourage many users to use this model. Can't we put the code you added to the test file to donwload the .so in a new utils file in the Mamba modeling folder? And then we could call it here and in the test.

Reply:
This is just readme file.

Reply:
I'm not sure what you mean. I think automatically downloading these .so files in the script if the model type is mamba would give a better user experience. Why not doing it there?

Reply:
@zzhang37  it seems the mamba run code fails if we don't set this environment variable at all. Is this intended? Can customer run with/without custom code? If the intention is always run mamba with custom kernel, we should put all these download/setup code in modeling_mamba file at the beginning before you load the library.

Reply:
@regisss Removed the all wget and added util to do the job.
@jiminha Just checked with latest build 538, works fine.